### PR TITLE
Creating a default Pull Request Template

### PR DESCRIPTION
**Pull request by alejandrovelez7**
### Context

This PR proposes a default PR template for all repositories in beamtech. 

This follows the instructions at -> https://help.github.com/en/articles/creating-a-default-community-health-file-for-your-organization

## Testing

Planning on merging this and seeing if a new pull request in `js-shared-configs` picks up the new default template.

### CODE REVIEW

**Comment by alexford** [comment]
I did not know `.github` was a thing! That's cool

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
-# .github
-Default community health files
+# Default Community Health Files
+
+These files help standardize pull request templates, issue templates, and potential code of conduct
+and contributing documents for all repos in beamtech.
```
Comment (alexford) [inline README.md]:
```markdown
This repo intends to standardize:

- Pull Request templates
- Issue templates
- Contributing documentation
- Code of conduct documentation
```

Found the sentence a little tough to read

File: README.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
-# .github
-Default community health files
+# Default Community Health Files
+
+These files help standardize pull request templates, issue templates, and potential code of conduct
+and contributing documents for all repos in beamtech. 
+
+For more information reference: 
+https://help.github.com/en/articles/creating-a-default-community-health-file-for-your-organization
```
Comment (alexford) [inline README.md]: Might be nice to make this a link

```
For more information, reference GitHub's [documentation](https://help.github.com/en/articles/creating-a-default-community-health-file-for-your-organization) on `.github` repos and community health files
```

To render as:

> For more information, reference GitHub's [documentation](https://help.github.com/en/articles/creating-a-default-community-health-file-for-your-organization) on `.github` repos and community health files

**Comment by alejandrovelez7** [comment]
> Do these defaults apply only to new repos or is it retroactive to any repos that don't have these files?

My understanding is that they should apply to existing repos. But I've never used this feature before so 🤞

**Pull request merged**